include/deal.II/grid/tria.h

```diff
@@ -946,53 +945,60 @@ namespace internal
  * been called.
  *
  *
- * <h3>%Boundary approximation</h3>
+ * <h3>Describing curved geometries</h3>
  *
- * You can specify a class describing the boundary for each boundary component.
- * If a new vertex is created on a side or face at the boundary, this class is
- * used to compute where it will be placed. The manifold indicator of the face
- * will be used to determine the proper component. See Manifold for the details.
- * Usage with the Triangulation object is then like this (let @p Ball be a
- * class derived from Manifold<tt><2></tt>):
+ * deal.II implements all geometries (curved and otherwise) with classes
+ * inheriting from Manifold; see the documentation of Manifold, step-49, or
+ * the
+ * @ref manifold
+ * module for examples and a complete description of the algorithms. By
+ * default, all cells in a Triangulation have a flat geometry, meaning that
+ * all lines in the Triangulation are assumed to be straight. If a cell has a
+ * manifold_id that is not equal to zero then the Triangulation uses the
```

Comment:
is it zero, of `numbers::flat_manifold_id`?

Reply:
Yup